entries/SWC-135.md

```diff
@@ -0,0 +1,17 @@
+# Title
+Statements With No Effects
```

Comment:
Why only statements? You could have expressions that have no effects as well would they fall under a different category? Also let's consider declarations.

Reply:
Good point. Perhaps just, "Code with no effects".